### DISCUSSION TITLE
Change the "from" name for Confirmation E-mail

### DISCUSSION THREAD

**Discussion by molimat** [supabase/supabase#891] [Questions] [answered]
Hello,

I tried to find it on the docs but with no success. 

Is there anyway to change the sender e-mail address that people sees in their inbox? Currently my clients are seeing "app.supabase.io". How can I change it?

**Comment by silentworks** [comment, marked as answer]
I think if you setup your own SMTP server this should resolve this.

**Reply by molimat** [reply]
Is there any doc for that? I saw some configurations using GoTrue but what if I use Supabase-js?

**Comment by molimat** [comment]
any news on it? :)

**Reply by silentworks** [reply]
You can find this information under `Authentication/Settings`

https://user-images.githubusercontent.com/79497/112302574-2728d280-8c93-11eb-8016-dac8c8d5857a.mp4

**Reply by molimat** [reply]
SUPER! Thank you

**Reply by molimat** [reply]
I never saw this, omg shame on me
